templates/base.html.twig

```diff
@@ -45,7 +45,7 @@
 </head>
 
 <body class="govuk-template__body">
-<script>document.querySelector('body').classList.add('js-enabled');</script>
+{#<script>document.querySelector('body').classList.add('js-enabled');</script>#}
```

Comment:
Hi @fiinix ,

Do you know why this has been removed? I can't see it mentioned anywhere in the commit message, and I can see various references to this class remain in the compiled CSS for the site.

I can see this code has been replicated in the app.js file, but it only runs if jQuery has been included on the page (which doesn't appear a given?), and the app.js code will likely execute after the page has already rendered, which will cause the page appearance to "jump" when this class is added to the page body.